### PR TITLE
Create Fibonacci.py

### PR DESCRIPTION
**Pull request by fomalhauting**
Issue #121

### CODE REVIEW

File: edualgo/maths/Fibonacci.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import math
+
+def fib(n):
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
+    else:
+        return fib(n-1) + fib(n-2)
+print([fib(i) for i in range(5)])
```
Comment (Abhijit2505) [inline edualgo/maths/Fibonacci.py:10]: please remove this line

File: edualgo/maths/Fibonacci.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import math
+
```
Comment (Abhijit2505) [inline edualgo/maths/Fibonacci.py:2]: Add the solution of the problem in the following methods as well,
- Dynamic Programming

**Review by Abhijit2505 — APPROVED** [review]

File: edualgo/maths/Fibonacci.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+def fib(n):
+    if n == 0:
+        return 0
+    elif n == 1:
+        return 1
+    else:
+        return fib(n-1) + fib(n-2)
+print([fib(i) for i in range(5)])
+
+// Output
```
Comment (Abhijit2505) [inline edualgo/maths/Fibonacci.py]: Remove these lines

**Review by Abhijit2505 — CHANGES_REQUESTED** [review]
Please look into these

**Comment by fomalhauting** [comment]
@Abhijit2505 done, please take a look and let me know if it is okay, thanks

**Pull request merged**